load_data/load_coverage.py

```diff
@@ -2,6 +2,25 @@
 from gnomad_hail import *
 
 
+def get_coverage_expr(mt):
```

Comment:
So I like the spirit behind this, but I think it needs some documentation. Most notably, now over_50 does not include those samples that are over_100, right?

Reply:
that's a great point -- I'll edit so it has the same behavior as before!